lib/index.js

```diff
@@ -19,6 +19,11 @@
  *   You should typically pass `React.Fragment`.
  * @property {string|undefined} [prefix='h-']
  *   React key prefix.
+ * @property {boolean|undefined} [mapDepricatedTableCellAttrsToInlineCss=true]
```

Comment:
```suggestion
 * @property {boolean|undefined} [fixTableCellAlign=true]
```

The current name is a bit long. What about this?
It’s just one idea, maybe you have more/better ones!